File: Latihan_b/php/Product.php
<?php

class Product{
    private $price;
    private $idProduct;

    function __construct($price = null, $idProduct = null)
    {
        $this->price = $price;
        $this->idProduct = $idProduct;
    }
        
    function setPrice($price)
    {
        $this->price = $price;
    }
        
    function getPrice()
    {
        return $this->price;
    }

    function setIdProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }
    
    function getIdProduct()
    {
        return $this->idProduct;
    }
}

?>

File: Latihan_b/py/Product.py
class Product:
    __price = ""
    __idProduct = ""

    def __init__(self):
        self.__price = ""    
        self.__idProduct = ""

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

    def setIdProduct(self, idProduct):
        self.__idProduct = idProduct

    def getIdProduct(self):
        return self.__idProduct